.hashibot.hcl

```diff
@@ -0,0 +1,11 @@
+poll "closed_issue_locker" "locker" {
+    schedule = "0 50 13 * * *"
+    closed_for = "720h" # 30 days
+    max_issues = 50
```

Comment:
You can crank this number a bit, otherwise it will take many many many days/runs to burn down the backlog.
```
max_issues = 500
```